roles/openshift_health_checker/library/rpm_version.py

```diff
@@ -3,6 +3,8 @@
 Ansible module for rpm-based systems determining existing package version information in a host.
 """
 
+from six import string_types
```

Comment:
Ansible bundles `six`, so it is better to use the bundled version than depending on a standalone version being installed in each host:

```python
from ansible.module_utils.six import string_types
```